<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-09-10T10:56:25 IST" name="Default suite" finished-at="2024-09-10T10:56:38 IST" duration-ms="12687">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:Stepdefinitions.TestRunner@7c37508a]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2024-09-10T10:56:25 IST" name="Default test" finished-at="2024-09-10T10:56:38 IST" duration-ms="12687">
      <class name="Stepdefinitions.TestRunner">
        <test-method is-config="true" signature="setUpClass(org.testng.ITestContext)[pri:0, instance:Stepdefinitions.TestRunner@7c37508a]" started-at="2024-09-10T10:56:25 IST" name="setUpClass" finished-at="2024-09-10T10:56:26 IST" duration-ms="376" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@62e7dffa]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:Stepdefinitions.TestRunner@7c37508a]" started-at="2024-09-10T10:56:26 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2024-09-10T10:56:38 IST" duration-ms="12215" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["check login is successful with valid credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["login functionality"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[valid login expected [https://www.facebook.com] but found [https://www.facebook.com/login/?privacy_mutation_token=eyJ0eXBlIjowLCJjcmVhdGlvbl90aW1lIjoxNzI1OTQ1OTkzLCJjYWxsc2l0ZV9pZCI6MzgxMjI5MDc5NTc1OTQ2fQ%3D%3D&next]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: valid login expected [https://www.facebook.com] but found [https://www.facebook.com/login/?privacy_mutation_token=eyJ0eXBlIjowLCJjcmVhdGlvbl90aW1lIjoxNzI1OTQ1OTkzLCJjYWxsc2l0ZV9pZCI6MzgxMjI5MDc5NTc1OTQ2fQ%3D%3D&next]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at Stepdefinitions.Loginsteps.user_is_navigated_to_the_home_page(Loginsteps.java:43)
at âœ½.user is navigated to the home page(file:///C:/Users/marc/eclipse-workspace/Cucumberpgm/src/test/resources/Features/login.feature:9)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:Stepdefinitions.TestRunner@7c37508a]" started-at="2024-09-10T10:56:38 IST" name="tearDownClass" finished-at="2024-09-10T10:56:38 IST" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- Stepdefinitions.TestRunner -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
